packages/story-editor/src/components/library/karma/mediaTab.karma.js

```diff
@@ -73,11 +74,43 @@ describe('Library Media Tab', () => {
         up(),
       ]);
       expect(
-        fixture.screen.getByRole('menuitem', { name: 'Edit' })
+        fixture.screen.getByRole('menuitem', { name: 'Edit meta data' })
       ).toBeDefined();
       expect(
-        fixture.screen.getByRole('menuitem', { name: 'Delete' })
+        fixture.screen.getByRole('menuitem', { name: 'Delete from library' })
       ).toBeDefined();
+      expect(
+        fixture.screen.getByRole('menuitem', { name: 'Insert image' })
+      ).toBeDefined();
+      expect(
+        fixture.screen.getByRole('menuitem', { name: 'Add as background' })
+      ).toBeDefined();
+    });
+
+    it('should allow setting media as background from the dropdown menu', async () => {
+      const mediaItem = fixture.editor.library.media.item(0);
+      // Hover the media
+      await fixture.events.mouse.moveRel(mediaItem, 20, 20, { steps: 2 });
+      await waitFor(() =>
+        expect(
+          fixture.screen.getByRole('button', { name: 'More' })
```

Comment:
Since you want to await this, you can use `findByRole` which combines `getByRole` and `waitFor`. 
ie. 
```
expect( await fixture.screen.findByRole('button', { name: 'More' })).toBeDefined();
```